op-conductor/conductor/service.go

```diff
@@ -647,9 +647,13 @@ func (oc *OpConductor) action() {
 
 	oc.log.Debug("exiting action with status and error", "status", status, "err", err)
 	if err != nil {
-		oc.log.Error("failed to execute step, queueing another one to retry", "err", err, "status", status)
-		time.Sleep(oc.retryBackoff())
-		oc.queueAction()
+		select {
+		case <-oc.shutdownCtx.Done():
```

Comment:
After an error, the service would wait to retry before actually attending to a shutdown. It loops back and then immediately shuts down anyway.